GW2EIEvtcParser/EIData/CombatReplay/Decorations/GenericAttachedDecoration.cs

```diff
@@ -5,29 +5,11 @@ namespace GW2EIEvtcParser.EIData
 {
     internal abstract class GenericAttachedDecoration : GenericDecoration
     {
-        [Flags]
-        public enum SkillModeCategory : uint
-        {
-            NotApplicable = 0,
-            ShowOnSelect = 1 << 0,
-            ImportantBuffs = 1 << 1,
-            ProjectileManagement = 1 << 2,
-            Heal = 1 << 3,
-            Cleanse = 1 << 4,
-            Strip = 1 << 5,
-            Portal = 1 << 6,
-        }
-
 
         public Connector ConnectedTo { get; }
         public RotationConnector RotationConnectedTo { get; protected set; }
-
-        public SkillConnector Owner { get; private set; }
-        public SkillModeCategory SkillCategory { get; private set; }
-
-        public ParserHelper.Spec Spec { get; private set; }
-
-        public long SkillID { get; private set; }
+        public SkillConnector Owner;
```

Comment:
Isn't this attribute redundant as the AbstractSingleActor is in SkillModeDescriptor? Perhaps add directly the SkillConnector inside SkillModeDescriptor instead of storing an AbstractSingleActor?